### PR TITLE
Img dn #6829

### PR DESCRIPTION
**Pull request by pallavipannu**

### CODE REVIEW

**Comment by deepio-oc** [comment]
Could you provide summary of changes done/why we need these changes ?

**Comment by pallavipannu** [comment]
Summary:
1. /opt/dkube/output - not accessible for writing in notebooks.so changed the notebooks. 
2. Pipeline : auth_token and keyword arguments given instead of positional args.
   - Taken care when a user upload a already existing pipeline, creates a different version.

File: image-denoising/README.md [hunk 1]
```diff
@@ -11,7 +11,7 @@
  2. Click *+Code* button.
  3. Select Code source as Git.
  4. Enter a unique name say *Img-DN*
- 5. Paste link https://github.com/oneconvergence/dkube-examples.git in the URL text box and enter branch as pytorch.
+ 5. Paste link https://github.com/oneconvergence/dkube-examples.git in the URL text box and enter branch as img-dn.
```
Comment (deepio-oc) [inline image-denoising/README.md]: this should stay as pytorch correct ?

File: image-denoising/dkube-denoising-pipeline.ipynb [hunk 2]
```diff
@@ -2,21 +2,25 @@
...
-   "execution_count": 53,
+   "execution_count": null,
    "metadata": {},
    "outputs": [],
    "source": [
+    "!pip install kfp==1.4.0 kfp-server-api==1.2.0\n",
+    "\n",
     "import sys\n",
-    "!{sys.executable} -m pip install kfp >/dev/null"
+    "!{sys.executable} -m pip install kfp >/dev/nullimport os\n"
```
Comment (deepio-oc) [inline image-denoising/dkube-denoising-pipeline.ipynb]:
```
import sys
!{sys.executable} -m pip install kfp==1.4.0 kfp-server-api==1.2.0 >/dev/null

**Comment by hemanthravi** [comment]
@pallavi-pannu-oc 
Make sure the pipeline is working with 1. 2.1.7.0 and on 2. demo system (2.2.1.12) or 2.2.1.13

**Review by deepio-oc — APPROVED** [review]

**Pull request merged**